examples/example-tls/testdata/README.md

```diff
@@ -0,0 +1,29 @@
+## Key Generation
+### CA Key/Cert Generation
```

Comment:
Why do we need info from the CA other than the CA cert?

Reply:
You will need the ca.key when generating client.pem and server.pem, but will not be needed when running the greeters. Remove?

```
openssl x509 -req -in client.csr -CA ca.cert -CAkey ca.key -CAcreateserial -out client.pem -days 365 -sha256 -extfile certificate.conf -extensions req_ext

openssl x509 -req -in service.csr -CA ca.cert -CAkey ca.key -CAcreateserial -out service.pem -days 365 -sha256 -extfile certificate.conf -extensions req_ext
```

Reply:
In that case let's leave it. Thanks!